readme.md

```diff
@@ -68,10 +68,22 @@ You can [sign up](https://wordpress.slack.com/signup) using your WordPress.org u
 
 #### Meetings: What to expect
 
-Meeting times and days
-Wave in
-Topics discussed
-Open Floors
+
+The meeting is usually to connect about WordPress happenings throughout the week connected with hosting and to catch up on the status of the team's ongoing projects.
+The agenda mainly consists of the following topics:
+
+* Greetings
+* Highlights
+* Hosting Team tasks
+* Open Floor
+
+During the Greetings we do a “Wave in” to get an overview who is in for the meeting and check how everyone is doing. New attendees are welcomed to introduce themselves.
```

Comment:
Do you think we should include the standard greeting -- or that should be on a different page?

Reply:
Maybe we can have another page like a "Standard Meeting" Page where we keep track of all our standard greetings

Reply:
Yeah, I like that. Maybe "Running a Meeting" or something?